src/textual/message_pump.py

```diff
@@ -367,35 +381,52 @@ async def _process_messages_loop(self) -> None:
                 except MessagePumpClosed:
                     break
 
+            self._active_message = message
```

Comment:
Purely for debugging.  If a handler is stuck, we can see what message is being processed.